setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+import os.path
+import sys
+
+readme_path = os.path.join(os.path.dirname(__file__), 'README.md')
+if sys.hexversion < 0x03000000:
+    readme = open(readme_path).read()
+else:
+    readme = open(readme_path, encoding='utf-8').read()
+
+setup(
+    name = 'sphinx_fossasia_theme',
+    version = '0.0.1',
+    author = 'Ujjwal Bhardwaj',
+    author_email = 'ujjwalb1996@gmail.com',
+    url = 'https://github.com/fossasia/sphinx_fossasia_theme',
+    license = 'GNU',
+    description = 'A Sphinx theme specific to FOSSASIA\'s Projects',
+    packages = ['fossasia_theme'],
```

Comment:
I've updated the PR. 

Where should the `__init__.py` exist and what should it contain?